ipn/ipnlocal/profiles.go

```diff
@@ -452,20 +452,34 @@ var defaultPrefs = func() ipn.PrefsView {
 	prefs.LoggedOut = true
 	prefs.WantRunning = false
 
-	prefs.ControlURL = winutil.GetPolicyString("LoginURL", "")
+	if controlURL, err := winutil.GetPolicyString("LoginURL"); err == nil {
+		prefs.ControlURL = controlURL
+	} else {
+		prefs.ControlURL = ""
+	}
+
 	prefs.ExitNodeIP = resolveExitNodeIP(netip.Addr{})
 
 	// Allow Incoming (used by the UI) is the negation of ShieldsUp (used by the
 	// backend), so this has to convert between the two conventions.
-	prefs.ShieldsUp = winutil.GetPolicyString("AllowIncomingConnections", "") == "never"
-	prefs.ForceDaemon = winutil.GetPolicyString("UnattendedMode", "") == "always"
+	shieldsUp, err := winutil.GetPolicyString("AllowIncomingConnections")
+	if err != nil {
+		shieldsUp = ""
```

Comment:
shouldn't need this. The default convention in Go is that if something returns a non-nil error, the other values are zero. Is that not the case here?